travel/res_config.py

```diff
@@ -36,6 +38,26 @@ def get_basic_passenger_limit(ir_config_parameter_pool, cr, uid, context=None):
         return limit
 
 
+def get_alert_address(ir_config_parameter_pool, cr, uid, context=None):
+    context = context or {}
```

Comment:
```python

class model(orm.Model):
    _name = "some.model"

    def foo(self, cr, uid, ids, context=None):
        if context is None:
            context = {}
        self.bar(cr, uid, ids, context=context)
        if context.get('some_key'):
             # do some stuff

    def bar(self, cr, uid, ids, context=None):
        if context is None:
            context = {}

class model_sub(orm.Model):
    _inherit = "some.model"

    def foo(self, cr, uid, ids, context=None):
        if context is None:
            context = {}
        super(model_sub, self).foo(cr, uid, ids, context=context)

    def bar(self, cr, uid, ids, context=None):
        context = context or {}
        context['some_key'] = True
        super(model_sub, self).bar(cr, uid, ids, context=context)
```

The trace goes as follows with `context = {}`:
* A call to `model_sub.foo` with `id(context) == 1`
    * A call to `model.foo` with `id(context) == 1`
        * A call to `model_sub.bar` with `id(context) == 1`
            * `context = context or {}` creates a new dict and assigns it to context `id(context) == 2`
            * A call to `model_sub.bar` with `id(context) == 2`
* Return to `model.foo`
    * `id(context) == 1`
    * `context == {}`